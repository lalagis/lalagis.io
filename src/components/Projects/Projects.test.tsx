import { describe, expect, test } from "vitest"
import renderer from 'react-test-renderer'
import Projects, { ProjectItem } from '.'

describe('Projects', () => {
  test('load projects', () => {
    const component = renderer.create(<Projects />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})