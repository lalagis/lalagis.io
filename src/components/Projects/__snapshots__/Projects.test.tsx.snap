// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Projects > load projects 1`] = `
<div
  className="mt-15"
>
  <p
    className="flex items-center mb-4 font-semibold text-3xl text-emerald"
  >
    项目 Projects
  </p>
  <div>
    <div
      className="flex flex-col gap-4"
    >
      <div
        className="px-4 py-3 rounded-md bg-gray-50 transition-colors decoration-none hover:bg-gray-100 dark:bg-gray-50/10 dark:hover:bg-gray-50/20"
      >
        <div
          className="flex h-full items-center sm:justify-center flex-row"
        >
          <div
            className="mr-4 text-4xl"
          >
            <div
              className="i-fluent-emoji-flat:card-index"
            />
          </div>
          <div
            className="ml-1"
          >
            <div
              className="font-medium leading-relaxed"
            >
              个人站 2.0
            </div>
            <div
              className="op-50 font-normal text-sm"
            >
              简历与项目经历
            </div>
          </div>
          <div
            className="sm:flex flex-1 flex-row justify-end gap-x-3 hidden"
          >
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://docs.astro.build/en/getting-started/"
              target="_blank"
              title="astro"
            >
              <div
                className="i-simple-icons:astro text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://www.typescriptlang.org/docs/"
              target="_blank"
              title="typescript"
            >
              <div
                className="i-simple-icons:typescript text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://unocss.dev/"
              target="_blank"
              title="unocss"
            >
              <div
                className="i-simple-icons:unocss text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://iconify.design/docs/"
              target="_blank"
              title="iconify"
            >
              <div
                className="i-simple-icons:iconify text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://vitest.dev/"
              target="_blank"
              title="vitest"
            >
              <div
                className="i-simple-icons:vitest text-3xl"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        className="px-4 py-3 rounded-md bg-gray-50 transition-colors decoration-none hover:bg-gray-100 dark:bg-gray-50/10 dark:hover:bg-gray-50/20"
      >
        <div
          className="flex h-full items-center sm:justify-center flex-row"
        >
          <div
            className="mr-4 text-4xl"
          >
            <div
              className="i-fluent-emoji-flat:cherries"
            />
          </div>
          <div
            className="ml-1"
          >
            <div
              className="font-medium leading-relaxed"
            >
              校园荔枝地图
            </div>
            <div
              className="op-50 font-normal text-sm"
            >
              webgis空间数据可视化与增删改查
            </div>
          </div>
          <div
            className="sm:flex flex-1 flex-row justify-end gap-x-3 hidden"
          >
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://nuxt.com/docs"
              target="_blank"
              title="nuxt 3+vue 3+typescript"
            >
              <div
                className="i-simple-icons:nuxtdotjs text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://docs.mapbox.com/"
              target="_blank"
              title="mapbox"
            >
              <div
                className="i-simple-icons:mapbox text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://docs.python.org/3"
              target="_blank"
              title="python+robyn"
            >
              <div
                className="i-akar-icons:python-fill text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://www.postgresql.org/docs/current/index.html"
              target="_blank"
              title="postgres"
            >
              <div
                className="i-akar-icons:postgresql-fill text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://postgis.net/"
              target="_blank"
              title="postgis"
            >
              <div
                className="i-akar-icons:postgresql-fill text-3xl"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        className="px-4 py-3 rounded-md bg-gray-50 transition-colors decoration-none hover:bg-gray-100 dark:bg-gray-50/10 dark:hover:bg-gray-50/20"
      >
        <div
          className="flex h-full items-center sm:justify-center flex-row"
        >
          <div
            className="mr-4 text-4xl"
          >
            <div
              className="i-fluent-emoji-flat:pushpin"
            />
          </div>
          <div
            className="ml-1"
          >
            <div
              className="font-medium leading-relaxed"
            >
              深圳大学“五四评优”评审
            </div>
            <div
              className="op-50 font-normal text-sm"
            >
              后端开发 大文件静态资源流动
            </div>
          </div>
          <div
            className="sm:flex flex-1 flex-row justify-end gap-x-3 hidden"
          >
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://docs.strapi.io/dev-docs/intro"
              target="_blank"
              title="strapi 4"
            >
              <div
                className="i-simple-icons:strapi text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://www.typescriptlang.org/docs/"
              target="_blank"
              title="typescript"
            >
              <div
                className="i-simple-icons:typescript text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://graphql.org/"
              target="_blank"
              title="graphql"
            >
              <div
                className="i-simple-icons:graphql text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="http://nginx.org/en/docs/"
              target="_blank"
              title="nginx"
            >
              <div
                className="i-simple-icons:nginx text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://cloud.tencent.com/document/product/228"
              target="_blank"
              title="cdn+对象存储"
            >
              <div
                className="i-akar-icons:cloud text-3xl"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        className="px-4 py-3 rounded-md bg-gray-50 transition-colors decoration-none hover:bg-gray-100 dark:bg-gray-50/10 dark:hover:bg-gray-50/20"
      >
        <div
          className="flex h-full items-center sm:justify-center flex-row"
        >
          <div
            className="mr-4 text-4xl"
          >
            <div
              className="i-fluent-emoji-flat:cherry-blossom"
            />
          </div>
          <div
            className="ml-1"
          >
            <div
              className="font-medium leading-relaxed"
            >
              你来花开
            </div>
            <div
              className="op-50 font-normal text-sm"
            >
              婚恋见面web平台 微信公众号与微信支付能力
            </div>
          </div>
          <div
            className="sm:flex flex-1 flex-row justify-end gap-x-3 hidden"
          >
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://nextjs.org/docs"
              target="_blank"
              title="nextjs+react+typescript"
            >
              <div
                className="i-akar-icons:nextjs-fill text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://graphql.org/"
              target="_blank"
              title="graphql"
            >
              <div
                className="i-simple-icons:graphql text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://docs.nestjs.com/"
              target="_blank"
              title="nestjs"
            >
              <div
                className="i-simple-icons:nestjs text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://www.prisma.io/"
              target="_blank"
              title="prisma"
            >
              <div
                className="i-simple-icons:prisma text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://www.postgresql.org/docs/current/index.html"
              target="_blank"
              title="postgres"
            >
              <div
                className="i-akar-icons:postgresql-fill text-3xl"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        className="px-4 py-3 rounded-md bg-gray-50 transition-colors decoration-none hover:bg-gray-100 dark:bg-gray-50/10 dark:hover:bg-gray-50/20"
      >
        <div
          className="flex h-full items-center sm:justify-center flex-row"
        >
          <div
            className="mr-4 text-4xl"
          >
            <div
              className="i-fluent-emoji-flat:telephone"
            />
          </div>
          <div
            className="ml-1"
          >
            <div
              className="font-medium leading-relaxed"
            >
              深圳大学管院通讯录
            </div>
            <div
              className="op-50 font-normal text-sm"
            >
              小程序便利师生校友会交流联系 发布活动
            </div>
          </div>
          <div
            className="sm:flex flex-1 flex-row justify-end gap-x-3 hidden"
          >
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://zh.uniapp.dcloud.io/"
              target="_blank"
              title="uniapp+vue 3"
            >
              <div
                className="i-akar-icons:shipping-box-02 text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://www.typescriptlang.org/docs/"
              target="_blank"
              title="typescript"
            >
              <div
                className="i-simple-icons:typescript text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://graphql.org/"
              target="_blank"
              title="graphql"
            >
              <div
                className="i-simple-icons:graphql text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://tailwindcss.com/docs/installation"
              target="_blank"
              title="tailwindcss"
            >
              <div
                className="i-simple-icons:tailwindcss text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://docs.strapi.io/dev-docs/intro"
              target="_blank"
              title="strapi 4"
            >
              <div
                className="i-simple-icons:strapi text-3xl"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        className="px-4 py-3 rounded-md bg-gray-50 transition-colors decoration-none hover:bg-gray-100 dark:bg-gray-50/10 dark:hover:bg-gray-50/20"
      >
        <div
          className="flex h-full items-center sm:justify-center flex-row"
        >
          <div
            className="mr-4 text-4xl"
          >
            <div
              className="i-fluent-emoji-flat:water-pistol"
            />
          </div>
          <div
            className="ml-1"
          >
            <div
              className="font-medium leading-relaxed"
            >
              白板管理后台
            </div>
            <div
              className="op-50 font-normal text-sm"
            >
              管理员相关社区操作web面板
            </div>
          </div>
          <div
            className="sm:flex flex-1 flex-row justify-end gap-x-3 hidden"
          >
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://nuxt.com/docs"
              target="_blank"
              title="nuxt 3+vue 3"
            >
              <div
                className="i-simple-icons:nuxtdotjs text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://www.typescriptlang.org/docs/"
              target="_blank"
              title="typescript"
            >
              <div
                className="i-simple-icons:typescript text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://graphql.org/"
              target="_blank"
              title="graphql"
            >
              <div
                className="i-simple-icons:graphql text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://tailwindcss.com/docs/installation"
              target="_blank"
              title="tailwindcss"
            >
              <div
                className="i-simple-icons:tailwindcss text-3xl"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        className="px-4 py-3 rounded-md bg-gray-50 transition-colors decoration-none hover:bg-gray-100 dark:bg-gray-50/10 dark:hover:bg-gray-50/20"
      >
        <div
          className="flex h-full items-center sm:justify-center flex-row"
        >
          <div
            className="mr-4 text-4xl"
          >
            <div
              className="i-fluent-emoji-flat:television"
            />
          </div>
          <div
            className="ml-1"
          >
            <div
              className="font-medium leading-relaxed"
            >
              白板BlankSpaces 2.0
            </div>
            <div
              className="op-50 font-normal text-sm"
            >
              现运行版本社区 前端协作
            </div>
          </div>
          <div
            className="sm:flex flex-1 flex-row justify-end gap-x-3 hidden"
          >
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://zh.uniapp.dcloud.io/"
              target="_blank"
              title="uniapp+vue 3"
            >
              <div
                className="i-akar-icons:shipping-box-02 text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://www.typescriptlang.org/docs/"
              target="_blank"
              title="typescript"
            >
              <div
                className="i-simple-icons:typescript text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://graphql.org/"
              target="_blank"
              title="graphql"
            >
              <div
                className="i-simple-icons:graphql text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://tailwindcss.com/docs/installation"
              target="_blank"
              title="tailwindcss"
            >
              <div
                className="i-simple-icons:tailwindcss text-3xl"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        className="px-4 py-3 rounded-md bg-gray-50 transition-colors decoration-none hover:bg-gray-100 dark:bg-gray-50/10 dark:hover:bg-gray-50/20"
      >
        <div
          className="flex h-full items-center sm:justify-center flex-row"
        >
          <div
            className="mr-4 text-4xl"
          >
            <div
              className="i-fluent-emoji-flat:open-mailbox-with-raised-flag"
            />
          </div>
          <div
            className="ml-1"
          >
            <div
              className="font-medium leading-relaxed"
            >
              深圳大学丽湖校区活动室预约系统 2.0
            </div>
            <div
              className="op-50 font-normal text-sm"
            >
              新架构重构预约小程序系统
            </div>
          </div>
          <div
            className="sm:flex flex-1 flex-row justify-end gap-x-3 hidden"
          >
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://cn.vuejs.org/guide/introduction.html"
              target="_blank"
              title="vue 3"
            >
              <div
                className="i-akar-icons:vue-fill text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://www.typescriptlang.org/docs/"
              target="_blank"
              title="typescript"
            >
              <div
                className="i-simple-icons:typescript text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://graphql.org/"
              target="_blank"
              title="graphql"
            >
              <div
                className="i-simple-icons:graphql text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://zh.uniapp.dcloud.io/"
              target="_blank"
              title="uniapp"
            >
              <div
                className="i-akar-icons:shipping-box-02 text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://docs.strapi.io/dev-docs/intro"
              target="_blank"
              title="strapi 4"
            >
              <div
                className="i-simple-icons:strapi text-3xl"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        className="px-4 py-3 rounded-md bg-gray-50 transition-colors decoration-none hover:bg-gray-100 dark:bg-gray-50/10 dark:hover:bg-gray-50/20"
      >
        <div
          className="flex h-full items-center sm:justify-center flex-row"
        >
          <div
            className="mr-4 text-4xl"
          >
            <div
              className="i-fluent-emoji-flat:bullseye"
            />
          </div>
          <div
            className="ml-1"
          >
            <div
              className="font-medium leading-relaxed"
            >
              白板地图微服务
            </div>
            <div
              className="op-50 font-normal text-sm"
            >
              校园兴趣点标识共建
            </div>
          </div>
          <div
            className="sm:flex flex-1 flex-row justify-end gap-x-3 hidden"
          >
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://docs-v3.strapi.io/developer-docs/latest/getting-started/introduction.html"
              target="_blank"
              title="strapi 3.0"
            >
              <div
                className="i-simple-icons:strapi text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://www.typescriptlang.org/docs/"
              target="_blank"
              title="typescript"
            >
              <div
                className="i-simple-icons:typescript text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://cloud.tencent.com/document/product/1207"
              target="_blank"
              title="腾讯云轻量服务器"
            >
              <div
                className="i-akar-icons:cloud text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://docs.docker.com/"
              target="_blank"
              title="docker"
            >
              <div
                className="i-simple-icons:docker text-3xl"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        className="px-4 py-3 rounded-md bg-gray-50 transition-colors decoration-none hover:bg-gray-100 dark:bg-gray-50/10 dark:hover:bg-gray-50/20"
      >
        <div
          className="flex h-full items-center sm:justify-center flex-row"
        >
          <div
            className="mr-4 text-4xl"
          >
            <div
              className="i-fluent-emoji-flat:megaphone"
            />
          </div>
          <div
            className="ml-1"
          >
            <div
              className="font-medium leading-relaxed"
            >
              白板BlankSpace 1.0
            </div>
            <div
              className="op-50 font-normal text-sm"
            >
              万级校园社区 前端协作
            </div>
          </div>
          <div
            className="sm:flex flex-1 flex-row justify-end gap-x-3 hidden"
          >
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://react.dev/"
              target="_blank"
              title="react"
            >
              <div
                className="i-akar-icons:react-fill text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://www.typescriptlang.org/docs/"
              target="_blank"
              title="typescript"
            >
              <div
                className="i-simple-icons:typescript text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://docs.taro.zone/docs/"
              target="_blank"
              title="taro"
            >
              <div
                className="i-akar-icons:shipping-box-02 text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://graphql.org/"
              target="_blank"
              title="graphql"
            >
              <div
                className="i-simple-icons:graphql text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://dgraph.io/"
              target="_blank"
              title="dgraph"
            >
              <div
                className="i-simple-icons:dgraph text-3xl"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        className="px-4 py-3 rounded-md bg-gray-50 transition-colors decoration-none hover:bg-gray-100 dark:bg-gray-50/10 dark:hover:bg-gray-50/20"
      >
        <div
          className="flex h-full items-center sm:justify-center flex-row"
        >
          <div
            className="mr-4 text-4xl"
          >
            <div
              className="i-fluent-emoji-flat:closed-mailbox-with-raised-flag"
            />
          </div>
          <div
            className="ml-1"
          >
            <div
              className="font-medium leading-relaxed"
            >
              深圳大学丽湖校区活动室预约系统 1.0
            </div>
            <div
              className="op-50 font-normal text-sm"
            >
              无纸化办公 师生物业三方协同
            </div>
          </div>
          <div
            className="sm:flex flex-1 flex-row justify-end gap-x-3 hidden"
          >
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://react.dev/"
              target="_blank"
              title="react"
            >
              <div
                className="i-akar-icons:react-fill text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://docs.taro.zone/docs/"
              target="_blank"
              title="taro"
            >
              <div
                className="i-akar-icons:shipping-box-02 text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://www.typescriptlang.org/docs/"
              target="_blank"
              title="typescript"
            >
              <div
                className="i-simple-icons:typescript text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://cloud.tencent.com/document/product/382"
              target="_blank"
              title="腾讯云短信"
            >
              <div
                className="i-akar-icons:cloud text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://developers.weixin.qq.com/miniprogram/dev/wxcloud/basis/quickstart.html"
              target="_blank"
              title="微信云开发"
            >
              <div
                className="i-simple-icons:wechat text-3xl"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        className="px-4 py-3 rounded-md bg-gray-50 transition-colors decoration-none hover:bg-gray-100 dark:bg-gray-50/10 dark:hover:bg-gray-50/20"
      >
        <div
          className="flex h-full items-center sm:justify-center flex-row"
        >
          <div
            className="mr-4 text-4xl"
          >
            <div
              className="i-fluent-emoji-flat:shallow-pan-of-food"
            />
          </div>
          <div
            className="ml-1"
          >
            <div
              className="font-medium leading-relaxed"
            >
              丰鑫同城
            </div>
            <div
              className="op-50 font-normal text-sm"
            >
              同城交流发帖定位
            </div>
          </div>
          <div
            className="sm:flex flex-1 flex-row justify-end gap-x-3 hidden"
          >
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://developers.weixin.qq.com/miniprogram/dev/framework/quickstart/code.html#WXML-%E6%A8%A1%E6%9D%BF"
              target="_blank"
              title="WXML"
            >
              <div
                className="i-akar-icons:html-fill text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://developers.weixin.qq.com/miniprogram/dev/framework/quickstart/code.html#WXSS-%E6%A0%B7%E5%BC%8F"
              target="_blank"
              title="WXSS"
            >
              <div
                className="i-akar-icons:css-fill text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://developers.weixin.qq.com/miniprogram/dev/framework/quickstart/code.html#JS-%E9%80%BB%E8%BE%91%E4%BA%A4%E4%BA%92"
              target="_blank"
              title="JavaScript"
            >
              <div
                className="i-akar-icons:javascript-fill text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://developers.weixin.qq.com/miniprogram/dev/wxcloud/basis/quickstart.html"
              target="_blank"
              title="微信云开发"
            >
              <div
                className="i-simple-icons:wechat text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://git-scm.com/doc"
              target="_blank"
              title="git"
            >
              <div
                className="i-simple-icons:git text-3xl"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        className="px-4 py-3 rounded-md bg-gray-50 transition-colors decoration-none hover:bg-gray-100 dark:bg-gray-50/10 dark:hover:bg-gray-50/20"
      >
        <div
          className="flex h-full items-center sm:justify-center flex-row"
        >
          <div
            className="mr-4 text-4xl"
          >
            <div
              className="i-fluent-emoji-flat:bird"
            />
          </div>
          <div
            className="ml-1"
          >
            <div
              className="font-medium leading-relaxed"
            >
              个人博客 1.0
            </div>
            <div
              className="op-50 font-normal text-sm"
            >
              三剑客入门之作
            </div>
          </div>
          <div
            className="sm:flex flex-1 flex-row justify-end gap-x-3 hidden"
          >
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://developer.mozilla.org/zh-CN/docs/Web/HTML"
              target="_blank"
              title="HTML 5"
            >
              <div
                className="i-akar-icons:html-fill text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://developer.mozilla.org/zh-CN/docs/Web/CSS"
              target="_blank"
              title="CSS 3"
            >
              <div
                className="i-akar-icons:css-fill text-3xl"
              />
            </a>
            <a
              className="flex flex-col rounded-md p-2 transition-colors decoration-none bg-white hover:bg-gray-200 dark:bg-gray-50/20 dark:hover:bg-gray-50/30"
              href="https://developer.mozilla.org/zh-CN/docs/Web/JavaScript"
              target="_blank"
              title="JavaScript"
            >
              <div
                className="i-akar-icons:javascript-fill text-3xl"
              />
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
